package.json

```diff
@@ -7,49 +7,46 @@
     "url": "git+https://github.com/fecgov/openFEC.git"
   },
   "scripts": {
-    "build": "export NODE_OPTIONS=--openssl-legacy-provider && npm run format:lint && webpack --config webpack.production.config.js --colors",
-    "develop": "export NODE_OPTIONS=--openssl-legacy-provider && webpack-dev-server --config webpack.config.js --mode development",
+    "build": "webpack --config webpack.production.config.js",
+    "develop": "webpack-dev-server --config webpack.config.js --mode development",
     "format": "prettier --write 'src/**/*.js'",
     "lint": "eslint ./src/**/*.js",
     "format:lint": "npm run format && npm run lint"
   },
   "dependencies": {
     "@braintree/sanitize-url": "^6.0.0",
     "immutable": "^3.8.2",
-    "minimist":"^1.2.6",
+    "minimist": "^1.2.6",
     "prop-types": "^15.6.2",
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
-    "async":"^2.6.4",
```

Comment:
async package had been added here to force updated version back when it was required as part of another package. but since it's not used at all anymore, I have removed it

Reply:
same for other removed packages